### PR TITLE
upf_to_json python package

### PR DESCRIPTION
**Pull request by simonpintarelli**
Checklist

- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/upf_to_json/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+
+test:
+  imports:
+    - upf_to_json
+
+about:
+  home: https://github.com/simonpintarelli/upf_to_json
+  license: BSD
+  license_family: BSD
```
Comment (marcelotrevisani) [inline recipes/upf_to_json/meta.yaml]: Please, add the ``license_file``

Comment (simonpintarelli) [reply recipes/upf_to_json/meta.yaml]: Thanks. I've added the license file. All tests are passing now.

**Comment by simonpintarelli** [comment]
@marcelotrevisani I'd be happy if you could consider to merge this PR. Thanks!

File: recipes/upf_to_json/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - six
+
+test:
+  imports:
+    - upf_to_json
+
+about:
+  home: https://github.com/simonpintarelli/upf_to_json
+  license: BSD
```
[the remark this answers is not in the document]
Comment (simonpintarelli) [reply recipes/upf_to_json/meta.yaml]: Yes, it is BSD-3-Clause, I've corrected it.

**Pull request merged**